src/content/cloud/okteto-cli.mdx

```diff
@@ -76,6 +76,11 @@ In order to refer to images in the `build` section of the Okteto Manifest, you c
 - `${OKTETO_BUILD_<service>_REPOSITORY}`: the name of the image that was pushed (`namespace/_<service>`)
 - `${OKTETO_BUILD_<service>_IMAGE}`: the full image reference (using its SHA)
 
+`${OKTETO_BUILD_<service>_IMAGE}` is equivalent to any of the following:
```

Comment:
what do we mean by it being equivalent to both values? what determines when I get the first vs getting the one with okteto in the tag?